src/ima.js

```diff
@@ -447,6 +490,9 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
         this._contentSrc = selectedSource[0].url;
       }
     });
+    this.eventManager.listenOnce(this.player, this.player.Event.DURATION_CHANGE, () => {
```

Comment:
why once?

Reply:
Because I'm looking for the first loading only, after it won't be changed

Reply:
if it won't be changed the `DURATION_CHANGE` won't be triggered....